src/index.js

```diff
@@ -88,6 +92,9 @@ export default function alias(options = {}) {
         return filePath + '.js';
       }
 
+      const nodeModuleFile = resolveNodeModules(updatedId);
+      if (nodeModuleFile) return nodeModuleFile;
```

Comment:
As you can see. The plugin should resolve the file relative to project.
After that, If no file can be resolve, it try to resolve it with node_modules folder under `process.cwd()`.
So I think it should't cause compatibility issues.